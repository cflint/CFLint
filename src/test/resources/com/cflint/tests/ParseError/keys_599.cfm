<cfset var foo =
	{
		0: "priority",
		5: "status",
		10: "name",
		"abc": "date",
		"xyz": "category"
	}
>