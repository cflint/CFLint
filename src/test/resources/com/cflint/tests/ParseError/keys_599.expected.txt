{
  "version" : "",
  "timestamp" : 1568171471,
  "issues" : [ ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ ],
    "countBySeverity" : [ ]
  }
}